cms/templates/widgets/header.html

```diff
@@ -7,11 +7,12 @@
   from django.urls import reverse
   from django.utils.translation import ugettext as _
   from cms.djangoapps.contentstore import toggles
+  from cms.djangoapps.contentstore.utils import get_pages_and_resources_url
+  from openedx.core.djangoapps.discussions.config.waffle import ENABLE_PAGES_AND_RESOURCES_MICROFRONTEND
```

Comment:
Can you remove all the instances where this was being passed in the context? Since we are importing it directly in the template now